src/components/CompareResults/TableHeader.tsx

```diff
@@ -114,27 +108,19 @@ function FilterableColumn({
   };
 
   const hasFilteredValues = uncheckedValues && uncheckedValues.size;
-  const baseAriaLabelText = hasSort
-    ? 'Click to filter and sort values'
-    : 'Click to filter values';
   const buttonAriaLabel = hasFilteredValues
-    ? `${name} (${baseAriaLabelText}. Some filters are active.)`
-    : `${name} (${baseAriaLabelText})`;
+    ? `${name} (Click to filter values. Some filters are active.)`
+    : `${name} (Click to filter values)`;
 
   return (
     <>
       <Button
-        color='tableHeaderButton'
         {...bindTrigger(popupState)}
+        color='tableHeaderButton'
+        size='small'
```

Comment:
note: this makes the font size the same as for the columns without a button, I find that this looks better then.